.github/workflows/schedule.yml

```diff
@@ -0,0 +1,28 @@
+name: Runs e2e every day at 2pm
```

Comment:
This is great .. so simple
(name does not match reality tho 😛 )
Also, I think we should canvass opinion on RT channel .. I don't want this to get in the way of demos, so I think we may want to run it out-of-hours ... 
Could even co-ordinate this with support, who have the responsibility right now of ensuring the RT/RA are running
(ooh, that has given me an idea about some further work for our expanding team 😄)

Reply:
This is just a sample test  to see if it is going to work for every 10 min😅 . If that's the case, then I will update the schedule.yml and have the cron job run every day at 2pm 😃

Reply:
ok, def switch to Dev instance then 😅